### DISCUSSION TITLE
xcode tutoring

### DISCUSSION THREAD

**Discussion by ultimate-supreme-leader** [sst-inc/help#7] [Q&A - iOS] [answered]
If you're really busy, just ignore me!

I really don't understand Xcode at all so if anybody's free (for at least 30 minutes) and doesn't mind explaining things step-by-step, could you tutor me? If you are busy, dropping some video links or other resources would still help me a lot. Thank you and sorry for the inconvenience!

**Comment by iso4096** [comment]
https://codewithchris.com/xcode-tutorial/

Hope this helps.

**Reply by granwyntan** [reply]
Thanks for sharing too

**Comment by granwyntan** [comment, marked as answer]
Sure, would you like to schedule a meeting?

If not, here are some resources:
1. Swift Website: https://swift.org/
2. Xcode Tutorial: https://www.youtube.com/watch?v=_YvAUgV6GYM
3. Segues: https://www.makeschool.com/mediabook/oa/tracks/build-ios-apps/learn-how-to-build-make-school-notes-in-swift-4/intro-segues/
4. Timer: https://www.hackingwithswift.com/articles/117/the-ultimate-guide-to-timer
5. IBOutlets and IBActions: https://www.hackingwithswift.com/read/2/5/from-outlets-to-actions-creating-an-ibaction
6. Documentation on:
- UIButton: https://developer.apple.com/documentation/uikit/uibutton
- UILabel: https://developer.apple.com/documentation/uikit/uilabel
7. You could also rewatch the videos on Google Classroom
